src/Tickets/Commerce/Gateways/Stripe/Webhooks/Payment_Intent_Webhook.php

```diff
@@ -34,6 +34,18 @@ public static function handle( array $event, Status_Interface $new_status, \WP_R
 
 		if ( empty( $order ) ) {
 
+			if ( empty( $payment_intent['metadata'][ Payment_Intent::$tc_metadata_identifier ] ) ) {
+				$response->set_status( 200 );
+				$response->set_data( sprintf(
+				// Translators: %1$s is the event id and %2$s is the event type name.
+					__( 'Event %1$s was received and will not be handled because the Payment Intent %2$s does not refer to an Event Tickets transaction.', 'event-tickets' ),
+					esc_html( Arr::get( $event, 'id' ) ),
+					esc_html( $payment_intent_id )
+				) );
+
+				return $response;
```

Comment:
Also, are we sure we want to send $response back with `200` ? I thought in that case we are going to return `null`.

Reply:
Yes. If we're not handling we need to return 200 otherwise Stripe will keep marking it as a failed event.